index.html

```diff
@@ -722,8 +722,12 @@ <h2>
             From the perspective of a <a>document</a>, locking to the
             <a>default orientation</a> is equivalent to unlocking because it
             means that it no longer has a lock applied. However, this does not
-            mean that the <a>[[\defaultOrientation]]</a> only contains the item
-            <a>any</a>.
+            mean that the <a>[[\defaultOrientation]]</a> should only contain
+            the item <a>any</a>. The <a>default orientation</a> is
```

Comment:
Can I suggest we change this section to:

```
        <p>
            From the perspective of a <a>document</a>, locking to the
            <a>default orientation</a> is equivalent to unlocking because it
            means that it no longer has a lock applied. 
          </p>
          <p class="note" title="Who controls the default orientation?">
            This does not
            mean that the <a>[[\defaultOrientation]]</a> will only contain
            the item <a>any</a>. The <a>default orientation</a> is
            likely device-specific and <a>[[\defaultOrientation]]</a> could, for
            example, contain <a>portrait-primary</a> and/or
            <a>landscape-primary</a>. Alternatively, the user could restrict the default orientaiton to a specific
            orientation via some OS or browser level preference for accessiblity reasons.
          </p>
```

Reply:
I somewhat feel that this section remain too conversational, so I'm still inclined to make it into a note (as suggested above).  Also, we should probably add a link to the web manifest section here, as it's pertinent to setting the default orientation.

Reply:
Sorry, didn't see the note suggestion!  That makes sense, will do that and add in the web manifest section link